### PR TITLE
FOUR-19237: Cases list "Pagination"

### PR DESCRIPTION
**Pull request by henryjonathanquispe**
## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-19237

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

### CODE REVIEW

File: resources/jscomposition/base/table/Pagination.vue
Comment (fagubla) [inline resources/jscomposition/base/table/Pagination.vue:1]: this file is used in the section caseDetail

File: resources/jscomposition/system/table/Pagination.vue [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+          stroke="#9FA8B5"
+          stroke-width="1.25"
+          stroke-linecap="round"
+          stroke-linejoin="round" />
+      </g>
+    </svg>
+
+    <svg
+      :class="{
+        'hover:tw-cursor-pointer hover:tw-bg-gray-100': pageModel < pages-1
```
Comment (fagubla) [inline resources/jscomposition/system/table/Pagination.vue:84]: pages - 1

File: resources/jscomposition/system/table/Pagination.vue [hunk 2]
```diff
@@ -0,0 +1,238 @@
+<template>
+  <div
+    class="tw-flex tw-items-center tw-justify-start tw-space-x-2 tw-text-gray-500 tw-text-sm">
+    <svg
+      :class="{
+        'hover:tw-cursor-pointer hover:tw-bg-gray-100': pageModel >0
```
Comment (fagubla) [inline resources/jscomposition/system/table/Pagination.vue:6]: > 0

File: resources/jscomposition/system/table/Pagination.vue [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+          stroke="#9FA8B5"
+          stroke-width="1.25"
+          stroke-linecap="round"
+          stroke-linejoin="round" />
+      </g>
+    </svg>
+
+    <svg
+      :class="{
+        'hover:tw-cursor-pointer hover:tw-bg-gray-100 ': pageModel >0
```
Comment (fagubla) [inline resources/jscomposition/system/table/Pagination.vue:28]: > 0

File: resources/jscomposition/system/table/Pagination.vue [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+        :value="pageModel"
+        class="tw-block tw-w-10 tw-text-center tw-flex-1 tw-border-0 tw-bg-transparent tw-pl-1
+          focus-visible:tw-outline-none placeholder:tw-text-gray-400"
+        placeholder="1"
+        @change="$emit('go',parseInt($event.target.value))">
+    </div>
+
+    <svg
+      :class="{
+        'hover:tw-cursor-pointer hover:tw-bg-gray-100': pageModel < pages-1
```
Comment (fagubla) [inline resources/jscomposition/system/table/Pagination.vue:62]: pages - 1

File: resources/jscomposition/system/table/Pagination.vue [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+    label: "15 items",
+  },
+  {
+    value: 30,
+    label: "30 items",
+  },
+  {
+    value: 50,
+    label: "50 items",
+
```
Comment (fagubla) [inline resources/jscomposition/system/table/Pagination.vue:146]: remove blank line

File: resources/jscomposition/system/table/Pagination.vue
Comment (fagubla) [inline resources/jscomposition/system/table/Pagination.vue:1]: translation for 'items' is missing

File: resources/jscomposition/base/table/TCell.vue [hunk 6]
```diff
@@ -45,7 +46,17 @@ export default defineComponent({
       }
       return get(props.row, props.column?.field) || "";
     };
+
+    const getComponent = () => props.column.cellRenderer().component;
+
+    const getParams = () => {
+      console.log("BREAK POINT AQUIII renderer");
```
Comment (fagubla) [inline resources/jscomposition/base/table/TCell.vue]: remove console

**Review by fagubla — CHANGES_REQUESTED** [review]

File: resources/jscomposition/base/table/Pagination.vue
Comment (fagubla) [inline resources/jscomposition/base/table/Pagination.vue:1]: This file is used in Case Detail

**Review by fagubla — CHANGES_REQUESTED** [review]

File: resources/jscomposition/base/table/Pagination.vue
Comment (henryjonathanquispe) [reply resources/jscomposition/base/table/Pagination.vue:1]: The file has customizations for the platform

**Review by fagubla — APPROVED** [review]

**Pull request merged**